src/MultivariateStats.jl

```diff
@@ -317,7 +317,7 @@ For more information about the algorithm, see the paper by Li, Zhu and Ogihara,
     cov_w::CovarianceEstimator = MS.SimpleCovariance()
     cov_b::CovarianceEstimator = MS.SimpleCovariance()
     out_dim::Int     = 0::(_ ≥ 0)
-    regcoef::Float64    = 1e-6::(_ ≥ 0)
+    regcoef::Real    = 1e-6::(_ ≥ 0)
     dist::SemiMetric = SqEuclidean()
```

Comment:
Generally a concrete type is preferred for model structs. Do you have a reason for not using one here?

Reply:
Ok